ts/components/LeftPane.tsx

```diff
@@ -237,6 +237,7 @@ export class LeftPane extends React.Component<Props, any> {
               rowHeight={64}
               rowRenderer={this.renderRow}
               width={width}
+              autoHeight
```

Comment:
Linter is complaining, you might need to assign something to autoHeight?
```
/Users/travis/build/loki-project/loki-messenger/ts/components/LeftPane.tsx:240:15
ERROR: 240:15  jsx-boolean-value  Value must be set for boolean attributes
```

Reply:
Fixed it

Reply:
Awsome!